homeassistant/components/unifi/device_tracker.py

```diff
@@ -453,13 +308,36 @@ def async_update_state(self, event: ItemEvent, obj_id: str) -> None:
                 + description.heartbeat_timedelta_fn(self.controller, self._obj_id),
             )
 
+    @callback
+    def async_event_callback(self, event: Event) -> None:
+        """Event subscription callback."""
+        if event.mac != self._obj_id or self._ignore_events:
+            return
+
+        description = self.entity_description
+        assert isinstance(description.event_to_subscribe, tuple)
+        assert isinstance(description.event_is_on, tuple)
```

Comment:
Is the goal to raise an exception if either of these two lines fail? Maybe a typo?

Reply:
This is just to make mypy happy, you can't reach this code without them being set

Reply:
Revisited this and could do something better, thanks!